rust-src/concordium_base/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased changes
 
+## 3.1.2
```

Comment:
Can you please keep the Unreleased changes and not change the version number in this PR.

We do this when releasing.